browser/brave_drm_tab_helper.cc

```diff
@@ -42,6 +46,15 @@ void BraveDrmTabHelper::OnWidevineKeySystemAccessRequest() {
 
   Browser* browser = chrome::FindBrowserWithWebContents(web_contents());
   browser->window()->UpdateToolbar(web_contents());
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+  // Show permission request bubble because showing blocked image alone in
+  // omnibox is not obvious to the user and easy to miss.
+  if (ShouldShowWidevineOptIn() && !is_widevine_permission_requested_) {
+    RequestWidevinePermission(web_contents());
+    is_widevine_permission_requested_ = true;
```

Comment:
Perhaps `is_widevine_permission_requested_` needs to be reset to `false` in `BraveDrmTabHelper::DidStartNavigation` just like `is_widevine_requested_` is? Currently, the popup only shows up once per tab. I would expect it to show up every time I navigate to a site that needs Widevine since the popup is tied to a specific site (e.g. "www.hulu.com wants to Install Widevine"). In my testing I navigated in the same tab to hulu.com (the popup shows) and then to netflix.com - the popup doesn't show.

Reply:
Though, I suppose this would make it popup for the same site if you don't install as well. So, maybe another mechanism?

Reply:
Oh,, thanks for checking.
I only want to prevent requesting widevine permission again for same site.
And I forgot WebContents is not changed when mainframe is replaced.
Fixed. by monitoring `RenderFrameCreated()` callback.